packages/core/src/components/toolbar/toolbar.ios.scss

```diff
@@ -61,7 +61,9 @@
 .header-translucent-ios .toolbar-background-ios,
 .footer-translucent-ios .toolbar-background-ios,
 .toolbar-translucent-ios .toolbar-background-ios {
-  background: rgba($toolbar-ios-background, $toolbar-ios-translucent-opacity);
+  // @color-mod in the future we will want to replace this with the following color-mod
+  // background-color: color-mod($toolbar-ios-background-color, a($alpha-ios-highest));
```

Comment:
Same with this comment - maybe we should document these things internally?

Reply:
Whats the best way to do this? I have comments throughout the SCSS prefixed with @color-mod in all the places we need to swap code when color-mod becomes an option.

Reply:
Oh I only saw it here. Maybe let's just add a `TODO` in front so we know that it's something we need to eventually do.